pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/JuniperShipKitRoute.java

```diff
@@ -0,0 +1,320 @@
+                                 @NonNull KitRequest kit, @NonNull KitRequestSettings kitRequestSettings,
+                                 @NonNull EasyPostUtil easyPostUtil, @NonNull String userIdRequest, @NonNull String kitTypeName,
+                                 String collaboratorParticipantId, String errorMessage, boolean uploadAnyway,
+                                 List<KitRequest> duplicateKitList, ArrayList<KitRequest> orderKits, List<KitRequest> specialKitList,
+                                 Value behavior, String externalOrderNumber, String uploadReason, String carrier) {
+        //TODO Pegah do we need kit special behavior checks? If so, how are we sending the response?
+//        if (behavior != null && StringUtils.isNotBlank(ddpInstance.getParticipantIndexES()) && !uploadAnyway) {
+////            Map<String, Map<String, Object>> participants =
+////                    ElasticSearchUtil.getFilteredDDPParticipantsFromES(ddpInstance, ElasticSearchUtil.BY_GUID + kit.getParticipantId());
+////            Map<String, Object> participant = participants.get(kit.getParticipantId());
+////            boolean specialKit = InstanceSettings.shouldKitBehaveDifferently(participant, behavior);
+////            if (specialKit) {
+////                if (InstanceSettings.TYPE_ALERT.equals(behavior.getType())) {
+////                    specialKitList.add(kit);
+////                } else if (InstanceSettings.TYPE_NOTIFICATION.equals(behavior.getType())) {
+////                    String message = "Kit uploaded for participant " + kit.getParticipantId() + ". \n" + behavior.getValue();
+//////                    notificationUtil.sentNotification(ddpInstance.getNotificationRecipient(), message,
+//////                            NotificationUtil.UNIVERSAL_NOTIFICATION_TEMPLATE, NotificationUtil.DSM_SUBJECT);
+////                } else {
+////                    logger.error("Instance settings behavior for kit was not known " + behavior.getType());
+////                }
+////            } else {
+////                //check with ddp_participant_id if participant already has a kit in DSM db
+////                handleKit(conn, ddpInstance, kitType, kit, kitRequestSettings, easyPostUtil, userIdRequest, kitTypeName,
+////                        collaboratorParticipantId, errorMessage, uploadAnyway, duplicateKitList, orderKits, externalOrderNumber,
+////                        uploadReason, carrier);
+////            }
+//        } else {
+            handleKit(conn, ddpInstance, kitType, kit, kitRequestSettings, easyPostUtil, userIdRequest, kitTypeName,
+                    collaboratorParticipantId, KitUtil.IGNORE_AUTO_DEACTIVATION, uploadAnyway, duplicateKitList, orderKits,
+                    externalOrderNumber, uploadReason, carrier);
+//        }
+    }
+
+    private void handleKit(@NonNull Connection conn, @NonNull DDPInstance ddpInstance, @NonNull KitType kitType, @NonNull KitRequest kit,
```

Comment:
When it is more figured out of what is and is not needed for Juniper, we can merge some of these handleKit methods into one